code/modules/urist/loadout/loadout_uniform_urist.dm

```diff
@@ -0,0 +1,15 @@
+/datum/gear/uniform/urist/dress
+	display_name = "fancy dress selection"
+	path = /obj/item/clothing/under
+
+/datum/gear/uniform/urist/dress/New()
```

Comment:
i guess we should just take the other dresses out of the loadout lists and put them all here